src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -530,12 +530,9 @@ protected override void EndProcessing()
                         _chosenListItems[i] = _chosenListItems[j];
```

Comment:
We could remove this too.

Reply:
No, that's how it's set.

Reply:
My apologies, I just realized what you meant to say. I think we should keep it, since it makes it clearer. Infact the algorithm can stop at i = n - 2.

Reply:
Why? It is dummy operation. For clarity, we just need to add a comment like "pick up random item j from current set (i,n), output it and remove it from the set by coping item i to j, continue with rest of the set (i+1, n)".

Reply:
Addressed changes. Please review the comments. Thanks.